src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/HttpsTests.cs

```diff
@@ -119,7 +125,7 @@ public static void SameBinding_Soap11_EchoString()
     }
 
     // Client and Server bindings setup exactly the same using Soap12
-    [Fact]
+    //[Fact]
```

Comment:
Nope, this was a mistake. When debugging xunit tests in VS, it's nice to only run the single test. Without a working VS test runner for corefx, this is the best that can be done.